content/en/blog/_posts/2023-04-14-improving-support-go-updates.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: blog
+title: "Improving Support: Go Updates"
+date: 2023-04-14
+slug: improving-support-go-updates
+---
+
+**Author**: [Jordan Liggitt](https://github.com/liggitt) (Google)
+
+### The Problem
+
+Since v1.19 (released in 2020), the Kubernetes project provides 12-14 months of patch releases for each minor version.
+This enables users to qualify and adopt Kubernetes versions in an annual upgrade cycle and receive security fixes for a year.
+
+The [go project](https://github.com/golang/go/wiki/Go-Release-Cycle#release-maintenance) releases new minor versions twice a year,
```

Comment:
Throughout the doc:

s/go project/Go project/
s/go version/Go version/
s/versions of go/versions of Go/
s/go patch/Go patch/
s/go minor/Go minor/
s/go team/Go team/
s/go security/Go security/
s/go user/Go user/
s/go community/Go community/

s/go1\./Go 1./